easybuild/easyconfigs/p/pyparsing/pyparsing-3.1.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'pyparsing'
+version = '3.1.1'
+
+homepage = 'https://github.com/pyparsing/pyparsing'
+description = """The pyparsing module is an alternative approach to creating and
+executing simple grammars, vs. the traditional lex/yacc approach, or the use of
+regular expressions. The pyparsing module provides a library of classes that
+client code uses to construct the grammar directly in Python code."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+# matplotlib 3.7.2 requires pyparsing<=3.1, this v3.1.1 will not works with it
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkg-config', '0.29.2', '', SYSTEM),
```

`pkg-config` is no longer actively maintained, so please use `pkgconf` instead
```suggestion
    ('pkgconf', '1.9.5'),
```